Tests/test_file_webp_animated.py

```diff
@@ -15,10 +15,7 @@
     skip_unless_feature,
 )
 
-pytestmark = [
-    skip_unless_feature("webp"),
-    skip_unless_feature("webp_anim"),
-]
+pytestmark = [skip_unless_feature("webp")]
```

Comment:
```suggestion
pytestmark = skip_unless_feature("webp")
```

This doesn't need to be a list if there is only one.